playbooks/installer/ospd/overcloud/post.yml

```diff
@@ -13,6 +13,24 @@
             dest: "{{ inventory_dir }}/id_rsa_overcloud"
             flat: yes
 
+- name: Generate ansible.ssh.config file
+  hosts: localhost
+  roles:
+    - role: common/ansible-ssh-config
+      when: installer.overcloud.network.isolation.enable == "yes"
+      external_network: "{{ installer.overcloud.network.template.content.parameter_defaults.ExternalNetCidr }}"
```

Comment:
Instead of duplicating the role invocation, use [ternary filter](http://docs.ansible.com/ansible/playbooks_filters.html#other-useful-filters) and/or `default(omit)` [filter](http://docs.ansible.com/ansible/playbooks_filters.html#omitting-undefined-variables-and-parameters) for `external_network`